api/src/db/models/base.py

```diff
@@ -9,14 +9,15 @@
 from sqlalchemy.orm import DeclarativeBase, Mapped, declarative_mixin, mapped_column
 from sqlalchemy.sql.functions import now as sqlnow
 
+from src.constants.schema import Schemas
 from src.util import datetime_util
 
 # Override the default naming of constraints
 # to use suffixes instead:
 # https://stackoverflow.com/questions/4107915/postgresql-default-constraint-names/4108266#4108266
 metadata = MetaData(
     naming_convention={
-        "ix": "%(column_0_label)s_idx",
+        "ix": "%(table_name)s_%(column_0_name)s_idx",
```

Comment:
This config details how SQLAlchemy + Alembic name indexes (and other constraints) in our database. The way it was named before would try adding the schema into the name, which unfortunately now did not match what we already had. This would have required Alembic to drop and recreate our indexes just to make the names match. I preferred to leave them alone and instead just adjust the naming to not include that.